stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/adapter/view/MessageReplyView.kt

```diff
@@ -153,27 +153,25 @@ public class MessageReplyView : FrameLayout {
                     }
                     setTint(color)
                 }
-                quotedMessage.isMine(ChatClient.instance().getCurrentUser()) -> {
+                isMine -> {
                     paintStyle = Paint.Style.FILL_AND_STROKE
-                    val color = if (isMine) {
-                        style?.messageBackgroundColorTheirs ?: context.getColorCompat(R.color.stream_ui_white)
-                    } else {
-                        style?.messageBackgroundColorMine ?: context.getColorCompat(R.color.stream_ui_grey_whisper)
-                    }
-                    setTint(color)
-                    style?.messageStrokeColorMine?.let(::setStrokeTint)
+                    setStrokeTint(
```

Comment:
`isMine` - is the flag which determines if `quotedMessage` is being located inside mine or their container message. This is the only parameter which we should take into account when applying styling